src/api/msServices/account/mfa/mfa.v6.service.js

```diff
@@ -0,0 +1,34 @@
+angular.module('ovh-api-services').service('OvhApiMsServicesAccountMfaV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiMsServicesAccountMfaV6');
```

Comment:
Since you've not implemented an `interceptor`, is `$cacheFactory` is needed in your case?

Reply:
No, removed.